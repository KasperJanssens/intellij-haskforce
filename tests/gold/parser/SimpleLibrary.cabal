library
  default-language:  Haskell2010

