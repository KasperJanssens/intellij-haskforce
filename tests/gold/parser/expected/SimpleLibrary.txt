Cabal File
  KEY_OR_CONFIG
    CONFIG
      LIBRARY
        PsiElement(LIBRARY)('library')
        BUILD_INFORMATION
          PsiElement(CabalTokenType.ld-options)('ld-options')
          PsiElement(CabalTokenType.:)(':')
          VARID
            PsiElement(CabalTokenType.varidRegexp)('duif')